Podcast/Authentication.swift

```diff
@@ -41,31 +79,31 @@ class Authentication: NSObject, GIDSignInDelegate {
 
         GIDSignIn.sharedInstance().scopes.append(contentsOf: [profileScope, emailScope])
         facebookLoginManager = LoginManager()
+        facebookLoginManager.loginBehavior = .web
```

Comment:
i like this view much better than the old one!

Reply:
although kinda annoying to log in every time and not have Facebook remember

Reply:
Yea the app behavior (over login) is buggier which is why I chose this one. The annoying this is it not remembering your password. Let me play again with the other one but I was having issues when it wasn't web

Reply:
ok, I don't imagine many people signing out of the app too often so it shouldn't be a super big issue